.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -112,6 +112,26 @@ body:
       render: shell
```

Comment:
Thanks @morganchen12 

> Devs using SPM through Xcode may not know how to find their Package.resolved (in Xcode 14, it's at `Project.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved`), so we'll likely need to add this to a playbook somewhere. Or maybe directly to this template?

This is new behavior, right? Is the Package.resolved not dumped in the same directory as the Package.swift?

Reply:
> This is new behavior, right?

Pretty sure it is.

Reply:
Thanks @morganchen12 and @peterfriese. 

I'll see if devs are including it or not in their new issues and go from there. 

I wonder what the motivation was to hiding the Package.resolved in .build/... I think I prefer it in the same directory at the Package.swift.